packages/eslint-config-react-app/index.js

```diff
@@ -23,6 +23,11 @@
 // To use them, explicitly reference them, e.g. `window.name` or `window.status`.
 const restrictedGlobals = require('confusing-browser-globals');
 
+
+// If we detect the project is using Flow, the ESLint rules are going to adapt
+// to improve the developer experience
+const isFlowProject = require('fs').existsSync('.flowconfig');
```

Comment:
This is probably not the right way to do it, I'm not sure this check is always going to refer to the root directory of the project. Any better idea? Or is it fine as is?